info/guide/guide-chapter-1.md

```diff
@@ -66,6 +66,7 @@ Note that prefixing can also negatively affect the style, as well as the auto ge
 <!-- =============================================================================== -->
 ### <a name="c1.2"></a> 1.2 Packages names
 ### <a name="r1.2.1"></a> Rule 1.2.1: Package names are in lower case and separated by dots. Code developed within your company should start with `your.company.domain`, and numbers are permitted in package names.
+Each file should have a `package` directive.
```

Comment:
Will you update full guide in this PR too?

Reply:
no